src/modules/claim/services/claim.service.ts

```diff
@@ -269,6 +269,35 @@ export class ClaimService {
     return qb.getMany();
   }
 
+  /**
+   * Get claims able to be revoked by user with matching DID
+   * @param filters additional filters
+   * @param currentUser revoker's DID
+   */
+  async getByRevoker({
+    currentUser,
+    filters: { namespace } = {},
+  }: {
+    filters?: QueryFilters;
+    currentUser: string;
+  }) {
+    let allRolesByRevoker = await this.rolesByRevoker(currentUser);
+    if (namespace) {
+      allRolesByRevoker = allRolesByRevoker?.filter(
+        (role) => role.namespace === namespace
+      );
+    }
+    const rolesByRevoker = allRolesByRevoker?.map((r) => r.namespace);
+    if (rolesByRevoker.length === 0) {
+      return [];
+    }
```

Comment:
```suggestion

```

Reply:
Just curious why we remove? The method for getting claims by issuer has this logic?

Reply:
No, i move it more upper. see other changes